.github/workflows/pr-tests_eslint-plugin-calcite-components.yml

```diff
@@ -0,0 +1,35 @@
+name: Run eslint-plugin-calcite-components tests
+
+on:
+  pull_request:
+    paths:
+      - "packages/eslint-plugin-calcite-components/**"
+    branches:
+      - master
+  push:
+    paths:
+      - "packages/eslint-plugin-calcite-components/**"
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install, build, and test
+        run: |
+          npm install
+          npm run build --if-present
```

Comment:
You can add the `--workspace=packages/eslint-plugin-calcite-components` flag here too, unless building CC and CCR is required for the eslint plugin's tests. That should fix the error in the action's error. 

Also we don't need the `--if-present` flag when running `build` from the root, because turbo will only run a package's `build` script if it is present.